config/routes.rb

```diff
@@ -116,6 +116,10 @@
     member do
       get :schedule
     end
+
+    member do
+      get :allevents
```

Comment:
@Ana06 https://en.wikipedia.org/wiki/Semantic_URL#Slug

Anyway, I still don't like the URL ```allevents```